### PR TITLE
JBIDE-20428instead of using JOB_NAME, use {snapshotLocation}-site to define the usage/installs/.../ tracker URL, eg., p2.statsURI = http://download.jboss.org/jbosstools/usage/installs/earlyaccess-site/4.3.0-SNAPSHOT/CR1/CR1-v20150811-1646/

### PR DESCRIPTION
**Pull request by nickboldt**

### CODE REVIEW

File: aggregate/component-site/pom.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
     <jbosstools-site>http://download.jboss.org/jbosstools/updates/stable/luna/</jbosstools-site>
     <jbosstools-tests-site>http://download.jboss.org/jbosstools/updates/nightly/coretests/master/</jbosstools-tests-site>
     <update.site.name>JBoss Tools - ${name}</update.site.name>
-    <snapshotLocation>components</snapshotLocation>
+    <snapshotLocation>component</snapshotLocation>
```
Comment (dgolovin) [inline aggregate/component-site/pom.xml]: Why this change is necessary? Is this snapshotLocation prope used somewhere?

Comment (nickboldt) [inline aggregate/component-site/pom.xml]: Not necessary. Just makes this consistent with the other places the variable is used, relative to the folder name. Eg, coretests-site/pom.xml  has snapshotLocation = coretests.
